tls/s2n_handshake_io.c

```diff
@@ -1038,6 +1038,15 @@ static int s2n_try_delete_session_cache(struct s2n_connection *conn)
     return 0;
 }
 
+S2N_RESULT s2n_wipe_record(struct s2n_connection *conn)
+{
```

Comment:
Could you add unit tests for this wipe function?

Reply:
It's a three line method with no branching that's hit by every call to s2n_negotiate. I don't think those tests would provide any value. But I'll make the method static.